fastai/data_block.py

```diff
@@ -197,6 +197,20 @@ def random_split_by_pct(self, valid_pct:float=0.2, seed:int=None)->'ItemLists':
         cut = int(valid_pct * len(self))
         return self.split_by_idx(rand_idx[:cut])
 
+    def split_subsets(self, train_size:float, valid_size:float, seed=None) -> 'ItemLists':
+        """Split the items into train set with size `train_size * n` and valid set with size `valid_size * n`.
```

Comment:
Please limit the docstring to one line, then add any other information in the corresponding doc notebook (fastai style guide).

Reply:
Done. (I think, the notebook documentation system is still a bit strange to me. Do I have to strip the output? Does that happen automatically? CONTRIBUTING.md is not too clear on this.)